### PR TITLE
ROX-24573: Update confusing language about scanner v4 indexer

### PR DESCRIPTION
**Pull request by kcarmichael08**
Version(s):
4.5+

[Issue](https://issues.redhat.com/browse/ROX-26473)

Link to docs preview:

[82761--ocpdocs-pr.netlify.app/openshift-acs/latest/architecture/acs-architecture.html](https://82761--ocpdocs-pr.netlify.app/openshift-acs/latest/architecture/acs-architecture.html)
[82761--ocpdocs-pr.netlify.app/openshift-acs/latest/cloud_service/acscs-architecture.html](https://82761--ocpdocs-pr.netlify.app/openshift-acs/latest/cloud_service/acscs-architecture.html)

QE review: **ACS has no QE, approved by SME**
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by RTann — APPROVED** [review]

File: modules/acs-central-services-overview.adoc [hunk 1]
```diff
@@ -40,17 +40,17 @@ You can use the same Central instance to secure multiple {ocp} or Kubernetes clu
...
+* link:https://nvd.nist.gov/vuln/search[NVD]: This is used for various purposes such as filling in information gaps when vendors do not provide information. For example, Alpine does not provide a description, CVSS score, severity, or published date.
 +
 [NOTE]
 ====
 This product uses the NVD API but is not endorsed or certified by the NVD.
 ====
 * link:https://github.com/stackrox/stackrox/blob/master/scanner/updater/manual/vulns.go[StackRox]: The upstream StackRox project maintains a set of vulnerabilities that might not be discovered due to data formatting from other sources or absence of data.
 
-The Scanner V4 Indexer uses the following sources:
+The Scanner V4 Indexer uses the following files to index Red Hat containers:
```
Comment (stevsmit) [inline modules/acs-central-services-overview.adoc:53]: I'm not actually sure if this is a hard requirement: https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces

```suggestion
The Scanner V4 Indexer uses the following files to index Red{nbsp}Hat containers:
```

File: modules/acscs-central-overview.adoc [hunk 2]
```diff
@@ -35,17 +35,17 @@ It handles API interactions and user interface ({product-title-short} Portal) ac
...
+* link:https://nvd.nist.gov/vuln/search[NVD]: This is used for various purposes such as filling in information gaps when vendors do not provide information. For example, Alpine does not provide a description, CVSS score, severity, or published date.
 +
 [NOTE]
 ====
 This product uses the NVD API but is not endorsed or certified by the NVD.
 ====
 * link:https://github.com/stackrox/stackrox/blob/master/scanner/updater/manual/vulns.go[StackRox]: The upstream StackRox project maintains a set of vulnerabilities that might not be discovered due to data formatting from other sources or absence of data.
 
-The Scanner V4 Indexer uses the following sources:
+The Scanner V4 Indexer uses the following files to index Red Hat containers:
```
Comment (stevsmit) [inline modules/acscs-central-overview.adoc:48]: Same note here.

File: modules/acs-central-services-overview.adoc (hunk 1 above)
Comment (kcarmichael08) [reply modules/acs-central-services-overview.adoc:53]: I always forget this! Thanks :)

**Pull request merged**